config/settings.d/registration.yml.example

```diff
@@ -1,2 +1,3 @@
 ---
 :enabled: false
+:registration_url: https://loadbalancer.example.com
```

Comment:
The example file is deployed on every fresh installation via packaging. I'd make it a comment.
```suggestion
# :registration_url: https://loadbalancer.example.com
```